<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RosterMediaPane</class>
 <widget class="QWidget" name="RosterMediaPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>202</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label_29">
     <property name="text">
      <string>Image to be used as an icon for that locomotive; head to the right:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_28">
     <property name="text">
      <string>Image to be used as main image for this locomotive </string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2" colspan="2">
    <widget class="DropJLabel" name="lblMainImage">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Image to be used for this locomotive,  drop any image file in the frame&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="lblIconImage">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Image to be used as an icon for this locomotive, drop any image file in the frame, have locomotive head to the right (for later rotations)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>image icon</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_27">
     <property name="text">
      <string>Use this tab to add various media to your roster</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="btnRemoveIcon">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="btnRemoveImage">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_30">
     <property name="text">
      <string>Web reference:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QLineEdit" name="edWebReference">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;URL to any web page related to that locomotive&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropJLabel</class>
   <extends>QLabel</extends>
   <header>dropjlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
